packages/dds/tree/src/test/simple-tree/api/treeNodeApi.spec.ts

```diff
@@ -51,6 +51,7 @@ import { testSimpleTrees } from "../../testTrees.js";
 import { FluidClientVersion } from "../../../codec/index.js";
 import { ajvValidator } from "../../codec/index.js";
 import { TreeAlpha } from "../../../shared-tree/index.js";
+import { isStableId } from "@fluidframework/id-compressor/internal";
```

Comment:
Nit: this import should be moved up above the package-local ones

Reply:
if we care about that, we should enable the linter rule which can auto fix it.

Reply:
We have one, but apparently `tree` has not opted in 🤷‍♂️

Reply:
Correction: it has explicitly opted out. I think this should be fixed, but there has been very little traction getting tree's eslint config updated to line up with the rest of the repo.

Reply:
I fixed the import anyway. I'm making a PR to format the imports of at least the non test code in tree since its easy to do.